### PR TITLE
OW-105: Media processor GA

### PR DESCRIPTION
**Pull request by joliveraortega**
#### What is this PR doing?

Media processor GA first commit.

### CODE REVIEW

File: lib/src/utils/utils.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export function isSupported(): Promise<void> {
+    return new Promise<void>((resolve, reject) => {
+        if ((typeof MediaStreamTrackProcessor === 'undefined') || (typeof MediaStreamTrackGenerator === 'undefined')) {
+            reject('Your browser does not support the experimental MediaStreamTrack API for Insertable Streams of Media.');
```
Comment (gmininberg) [inline lib/src/utils/utils.ts:4]: why experimental?

Comment (joliveraortega) [reply lib/src/utils/utils.ts:4]: I took that string from the original text you had. See: 

Screenshot 2022-02-15 at 09 14 16

If you prefer another string please let me know.

Comment (gmininberg) [reply lib/src/utils/utils.ts:4]: this one was just copy paste that i got from somewhere... i think its just good to remove experimental

Comment (joliveraortega) [reply lib/src/utils/utils.ts]: Review comment addressed. Thanks.

**Pull request merged**